bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/api/dto/clip2/Event.java

```diff
@@ -33,6 +35,11 @@ public class Event {
     }.getType();
 
     private @Nullable List<Resource> data = new ArrayList<>();
+    private @Nullable @SerializedName("type") String contentType; // content type of resources
```

Comment:
I think this could be:
```suggestion
    private @Nullable @SerializedName("type") ContentType contentType; // content type of resources
```

please see other comments in `ContentType`.

Reply:
.. as above ..